CMakeLists.txt

```diff
@@ -579,10 +580,10 @@ endif()
 if(UNIX)
   if(APPLE)
     if (ARCH MATCHES "arm64")
-      set(ENERGYPLUS_EXPECTED_HASH 1faddd434694d18712147048444cb92c)
+      set(ENERGYPLUS_EXPECTED_HASH 4704fcc75312792fc9c4f5a9359fb3eb)
```

Comment:
this is definitely not right, since I haven't built the arm64 package yet :)

Reply:
This is not [EnergyPlus-22.2.0-1d22b9a61f-Darwin-macOS12.1-x86_64.tar.gz](https://github.com/NREL/EnergyPlus/releases/download/v22.2.0-RC1/EnergyPlus-22.2.0-1d22b9a61f-Darwin-macOS12.1-x86_64.tar.gz)?

Reply:
Oh, oops, you're right.